webapp/src/main/java/com/box/l10n/mojito/service/branch/DeleteBranchService.java

```diff
@@ -0,0 +1,44 @@
+package com.box.l10n.mojito.service.branch;
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+import com.box.l10n.mojito.entity.Branch;
+import com.box.l10n.mojito.service.asset.AssetService;
+import java.util.Set;
+import org.slf4j.Logger;
+import org.springframework.stereotype.Service;
+
+/**
+ * Service to manage deletion of {@link Branch}es.
+ *
+ * @author jeanaurambault
```

Comment:
this could be you, or removed :-)

Reply:
I actually thought about changing it, but the code is the same code written by you. It was just placed in another class by me